### DISCUSSION TITLE
使用Github Action自动化部署 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#162] [General]
# 使用Github Action自动化部署 - 愧怍的小站

如果有写过项目的经历，就免不了将代码上传到服务器上，安装依赖，然后输入启动命令的步骤。但是有的项目往往需要经常性的改动，如果还是照着上面的方式进行部署的话。先不说这样操作的效率，操作个几次就想罢工了。并且上面这样操作的往往容易误操作。而 Github Actions 正是该问题的良药。

https://kuizuo.cn/use-github-action-to-auto-deploy/#comments

**Comment by zkeq** [comment]
好文

**Reply by kuizuo** [reply]
感谢支持
